addons/website/views/website_templates.xml

```diff
@@ -1435,6 +1435,13 @@
     </xpath>
 </template>
 
+<!-- Area for blocks shared by all pages -->
+<template id="shared_blocks" inherit_id="website.layout" name="Shared blocks" active="True">
```

Comment:
Setting `active="True"`, IMO, suggests this is something that can be disabled by code / some option. Remove it.